validator/src/main.rs

```diff
@@ -1668,8 +1668,8 @@ pub fn main() {
         .arg(
             Arg::with_name("accounts_db_skip_shrink")
                 .long("accounts-db-skip-shrink")
-                .help("Enables faster starting of validators by skipping shrink. \
-                      This option is for use during testing."),
+                .help("This is obsolete since it is now enabled by default. Enables faster starting of validators by skipping startup clean and shrink.")
+                .hidden(true),
```

Comment:
i guess we don't have that generic deprecated args logic in 1.14?  mind adding something like 

```rust
if matches.is_present("accounts_db_skip_shrink") {
    warn!("`--accounts-db-skip-shrink` is deprecated. please consider removing it from the validator command line argument list");
}
```

just above where where the `ValidatorConfig` is built?